pyomeca/rototrans.py

```diff
@@ -4,7 +4,8 @@
 import pandas as pd
 import xarray as xr
 
-from .angles import Angles
+from pyomeca import Angles
```

Comment:
Does this may cause problem if ones has a pyomeca installed, but also has imported the folder (as a symbolic link for instance)?